interface/main/calendar/modules/PostCalendar/pnadmin.php

```diff
@@ -39,21 +40,20 @@ function postcalendar_admin_modifyconfig($msg = '', $showMenu = true)
 
     $output->SetInputMode(_PNH_VERBATIMINPUT);
 
-    $header = "<html><head><title>" . xlt("Calendar") . "</title></head>";
-    $header .= <<<EOF
-	
-	<body bgcolor=
-EOF;
-    $header .= '"' . attr($GLOBALS['style']['BGCOLOR2']) . '">';
+    $header = "<html><head><title>" . xlt("Calendar") . "</title>";
+    $header .= Header::setupHeader('', false)  . '</head><body>';
```

Comment:
I am not sure if passing `''` is necessary

Reply:
I am not sure too, But I just follow @bradymiller, sir tell me import bootstrap syntax.

https://github.com/openemr/openemr/pull/3069#pullrequestreview-369196139

Reply:
yep, that was my bad. Gonna likely bring this into codebase soon after review. I'll then do a quick commit to remove those.

Reply:
Thank you :)

Reply:
@im-Amitto , trying to remove the '' in the first parameter throws a parsing error:
```
Header::setupHeader(, false)
```
Since we have to set the second parameter to false, I think we are stuck with the `''`. Let me know if there's another way I'm not thinking of.

Reply:
I guess he just feel alone. let's give him his `''` or we can use `[]`which is his default value in case or we can go with `undefined`, so many fish in the sea.